app/modules/entities/components/layouts/works_browser_facets.svelte

```diff
@@ -1,31 +1,34 @@
 <script>
   import SelectDropdown from '#components/select_dropdown.svelte'
-  import { i18n } from '#user/lib/i18n'
-  import { entityProperties, getWorksFacets } from '#entities/components/lib/works_browser_helpers'
-  import Spinner from '#components/spinner.svelte'
+  import { entityProperties, getWorksFacets, getFacetsEntitiesBasicInfo } from '#entities/components/lib/works_browser_helpers'
   import { getContext } from 'svelte'
+  import { i18n, I18n } from '#user/lib/i18n'
 
   export let works, facets, facetsSelectors, facetsSelectedValues, flash
 
   const layoutContext = getContext('layout-context')
+  const facetsObj = getWorksFacets({ works, context: layoutContext })
 
-  const waitForFacets = getWorksFacets({ works, context: layoutContext })
+  let loadingMessage
+  const urisSize = facetsObj.valuesUris.length
```

Comment:
In this code base, "size" is used only for fonts and images so far (outside of Elasicsearch DSL), while arrays have "length" and "count". Unless there is a particular reason why "size" is a better fit?
```suggestion
  const urisCount = facetsObj.valuesUris.length
```

Reply:
That's a developer's habit coming from past inventaire life. I used both `size` and `count` interchangeably, but since it hurts your eyes, i changed it in e5a6b238a